include/vcpkg/base/messages.h

```diff
@@ -2091,5 +2091,168 @@ namespace vcpkg
+                    "",
+                    "Failed to parse serialized manifest file of {path} Please open an issue at "
+                    "https://github.com/microsoft/vcpkg, with the following output:");
+    DECLARE_MESSAGE(MismatchedSerializedManifestSCF,
+                    (),
+                    "",
+                    "The serialized manifest SCF was different from the original SCF. Please open an issue at "
+                    "https://github.com/microsoft/vcpkg, with the following output:");
+    DECLARE_MESSAGE(PortBugIncludeDirInCMakeHelperPort,
+                    (),
+                    "",
+                    "The folder /include exists in a cmake helper port; this is incorrect, since only cmake "
+                    "files should be installed");
+    DECLARE_MESSAGE(PortBugMissingIncludeDir,
+                    (),
+                    "",
+                    "The folder /include is empty or not present. This indicates the library was not correctly "
+                    "installed.");
+    DECLARE_MESSAGE(PortBugRestrictedHeaderPaths,
+                    (),
+                    "",
+                    "Restricted headers paths are present. These files can prevent the core C++ runtime and "
+                    "other packages from compiling correctly:");
+    DECLARE_MESSAGE(PortBugAllowRestrictedHeaders,
+                    (msg::env_var),
+                    "",
+                    "In exceptional circumstances, this policy can be disabled via {env_var}");
+    DECLARE_MESSAGE(PortBugDuplicateIncludeFiles,
+                    (),
+                    "'{CURRENT_PACKAGES_DIR}' should not be translated.",
+                    "Include files should not be duplicated into the /debug/include directory. If this cannot "
+                    "be disabled in the project cmake, use\n"
+                    "file(REMOVE_RECURSE \"${CURRENT_PACKAGES_DIR}/debug/include\")");
+    DECLARE_MESSAGE(PortBugDebugShareDir,
+                    (),
+                    "'{CURRENT_PACKAGES_DIR}' should not be translated.",
+                    "/debug/share should not exist. Please reorganize any important files, then use\n"
+                    "file(REMOVE_RECURSE \"${CURRENT_PACKAGES_DIR}/debug/share\")");
+    DECLARE_MESSAGE(PortBugMissingFile,
+                    (msg::path),
+                    "",
+                    "The /{path} file does not exist. This file must exist for CMake helper ports.");
+    DECLARE_MESSAGE(PortBugMergeLibCMakeDir,
+                    (msg::spec),
+                    "",
+                    "The /lib/cmake folder should be merged with /debug/lib/cmake and moved to "
+                    "/share/{spec}/cmake.");
+    DECLARE_MESSAGE(PortBugUseVcpkgCMakeConfigFixup,
+                    (),
+                    "",
+                    "Please use the helper function `vcpkg_cmake_config_fixup()` "
+                    "from the port vcpkg-cmake-config.`");
```

Comment:
Ditto I think these should be combined.

```suggestion
    DECLARE_MESSAGE(PortBugMergeLibCMakeDir,
                    (msg::spec),
                    "",
                    "The /lib/cmake folder should be merged with /debug/lib/cmake and moved to /share/{spec}/cmake. Please use the helper function `vcpkg_cmake_config_fixup()` from the port vcpkg-cmake-config.`");
```